### PR TITLE
add procstat processing

### PR DESCRIPTION
**Pull request by steverweber**
add support for processing /proc/stat

https://github.com/mozilla-services/heka/pull/1320

### CODE REVIEW

File: sandbox/lua/filters/procstat.lua [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+
+--[[
+
+Calculates deltas in /proc/stat data.
+Also emits CPU percentage utilization information.
+
+Config:
+
+- whitelist (string, optional, default nil)
+    Only proccess fields that fit the pattern, defaults to match all
```
Comment (acesaro) [inline sandbox/lua/filters/procstat.lua]: Spelling: "process" instead of "proccess"; there are a couple others like this in the file too.

**Comment by steverweber** [comment]
opps, ill fix that up now.

**Comment by trink** [comment]
The typos were never fixed.

**Comment by steverweber** [comment]
@trink , i took another look and did not find them... perhaps you could message in the code,

**Comment by steverweber** [comment]
Bla, sorry @trink , watching movie, /not all there/.  I can fix tomorrow or please feel free to fork and fix

**Comment by rafrombrc** [comment]
@trink I fixed them, edited the changelog, and integrated the docs into the right places before merging, see dda6279716c01a6545fc16e7b2416d7df244cba3.

**Comment by trink** [comment]
Interesting the PR in no way reflects that. 
+1

**Comment by rafrombrc** [comment]
Yeah, that's b/c I can't push my changes back up to @steverweber's branch, to which this PR is connected. But I'll often pull down a remote branch to a short lived branch on my local machine, make an additional commit to tweak some things, and then merge that branch back to dev and push. Github notices that all of the commits in the PR have landed and marks the PR as merged automatically.

**Comment by steverweber** [comment]
thanks for fixing that up.

**Pull request merged**